### PR TITLE
Add ERC: Contract wallet management NFT

### PR DESCRIPTION
**Pull request by wenzhenxiang**
The smart contract wallet interface of this ERC is for nft management and nft approval. this EIP is designed for all nft.

Ethereum-magicians [discussion](https://ethereum-magicians.org/t/erc-draft-contract-wallet-management-nft/16702)

### CODE REVIEW

File: ERCS/erc-x.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
+---
+eip: x
```
Comment (abcoathup) [inline ERCS/erc-x.md:2]:
```suggestion
eip: 7564
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

File: ERCS/erc-x.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+title: Contract wallet management nft
```
Comment (abcoathup) [inline ERCS/erc-x.md:3]:
```suggestion
title: Contract wallet management NFT
```

File: ERCS/erc-7564.md [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-11-21
+requires: 165
+---
+
+## Abstract
+
+This proposal introduces a smart contract wallet-based approach for managing nfts, focusing on utilizing the programmable features of smart contract wallets for nft asset management. Additionally, it introduces functions such as `nftApprove`, `nftSetApprovalForOneAll`, `nftSetApprovalForAllAll`, `nftGetApproved`, `nftIsApprovedForOneAll`, `nftIsApprovedForAllAll` and `nftTransfer`, which provide enhanced control over nft transactions. This approach seeks to enhance nft management by utilizing the built-in features of smart contract wallets, thus offering a more adaptable, secure, and efficient method for managing token transactions.
```
Comment (SamWilsn) [inline ERCS/erc-7564.md:16]:
```suggestion
This proposal introduces a smart contract wallet-based approach for managing NFTs, focusing on utilizing the programmable features of smart contract wallets for NFT asset management. Additionally, it introduces functions such as `nftApprove`, `nftSetApprovalForOneAll`, `nftSetApprovalForAllAll`, `nftGetApproved`, `nftIsApprovedForOneAll`, `nftIsApprovedForAllAll` and `nftTransfer`, which provide enhanced control over NFT transactions. This approach seeks to enhance NFT management by utilizing the built-in features of smart contract wallets, thus offering a more adaptable, secure, and efficient method for managing token transactions.
```

Please capitalize all initialisms throughout the document.

File: ERCS/erc-7564.md [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+
+
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+**Compliant contract must implement the [ERC-165](./eip-165) interfaces**
+```solidity
+// SPDX-License-Identifier: GPL-3.0
```
Comment (SamWilsn) [inline ERCS/erc-7564.md:46]: We cannot permit GPL'd code in the EIPs repository. Code inline in the document itself _must_ be CC0-1.0. I permit a few more licenses (`MIT`, `Apache-2.0`, etc.) in the assets directory, but again, CC0-1.0 is preferred.

Comment (wenzhenxiang) [reply ERCS/erc-7564.md]: thanks so much, I deleted it.
also, Happy Spring Festival

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**